lab_1/main.py

```diff
@@ -14,13 +14,11 @@ def tokenize(text: str) -> list or None:
 
     if not isinstance(text, str):
         return None
-    isalpha_text = ''
+    tokens = ''
     for i in text:
-        if i.isalpha() is True or i == ' ':
-            isalpha_text += i
-    isalpha_text = isalpha_text.lower()
-    tokens = isalpha_text.split()
-    return tokens
+        if i.isalpha() or i == ' ':
```

Comment:
if `i` is `'\n' `or` '\t'` or any white space character? Consider using `isspace`

Reply:
I used it after your first told me about this but I forgot to commit and push changes, I'm really sorry!